packages/jss/tests/integration/sheet.js

```diff
@@ -529,6 +529,36 @@ describe('Integration: sheet', () => {
       `)
     })
 
+    it('should correctly escape the name', () => {
+      const localJss = create({
+        createGenerateId() {
+          return (rule, sheet) => `${sheet.options.classNamePrefix}-${rule.key}-id`
```

Comment:
can you move it to the createGenerateId we use everywhere in tests? Seems like this could be just used there without any issues, I mean without changing the class name in all assertions.